libs/azure-iot-middleware-freertos/libraries/azure-sdk-for-c/sdk/src/azure/iot/az_iot_adu_ota.c

```diff
@@ -509,26 +507,9 @@ AZ_NODISCARD az_result az_iot_adu_ota_get_service_properties_response(
           iot_hub_client, &jw,
           AZ_SPAN_FROM_STR(AZ_IOT_ADU_OTA_AGENT_PROPERTY_NAME_SERVICE), status, version, AZ_SPAN_EMPTY));
 
+    // It is not necessary to send the properties back in the acknowledgement.
+    // We opt not to send them to reduce the size of the payload.
     _az_RETURN_IF_FAILED(az_json_writer_append_begin_object(&jw));
```

Comment:
is this empty object still needed? I forget what the `begin_response_status()` appends.

Reply:
Yes it's needed. Without that the property does not have an empty object. This is what that generates as of now:

    "deviceUpdate": {
        "__t": "c",
        "service": {
            "ac": 200,
            "av": 123,
            "value": {}
        }
    }

Reply:
Is `value` even needed?

Reply:
https://docs.microsoft.com/en-us/azure/iot-develop/concepts-convention#writable-properties
Looks like yes :)